profiles/Gulshankhandale.json

```
@@ -0,0 +1,6 @@
+{
+  "image": "https://i.ibb.co/t8J1vm3/profile.png",
```

It's better if you use a GitHub-hosted image, IMO.
This is the screenshot that's in the issue. #1442
```suggestion
  "image": "https://camo.githubusercontent.com/8176b8501fe493e161dcec9cbe659e5d3e9740bc8d41420f30db21cf24f2a511/68747470733a2f2f692e6962622e636f2f74384a31766d332f70726f66696c652e706e67",
```